bindings/mqtt/mqtt.go

```diff
@@ -186,14 +186,27 @@ func (m *MQTT) Operations() []bindings.OperationKind {
 }
 
 func (m *MQTT) Invoke(req *bindings.InvokeRequest) (*bindings.InvokeResponse, error) {
-	m.logger.Debugf("mqtt publishing topic %s with data: %v", m.metadata.topic, req.Data)
-
-	token := m.producer.Publish(m.metadata.topic, m.metadata.qos, m.metadata.retain, req.Data)
-	if !token.WaitTimeout(defaultWait) || token.Error() != nil {
-		return nil, fmt.Errorf("mqtt error from publish: %v", token.Error())
-	}
-
-	return nil, nil
+	// MQTT client Publish() has an internal race condition in the default autoreconnect config.
```

Comment:
Need to comment delete time when resiliency feature introduced.

Reply:
To clarify, is the recommendation to reword to:
> This is not configurable to keep this as an implementation detail for this component, as the additional public config metadata required **will** be replaced by the more general Dapr APIs for resiliency moving forwards.

Reply:
I'm fine with keeping this as is and for this components to be updated when resiliency features are out.